plugins/statistics.koplugin/main.lua

```diff
@@ -1058,6 +1069,10 @@ function ReaderStatistics:getPageTimeTotalStats(id_book)
     return total_pages, total_time
 end
 
+function ReaderStatistics:usePageMapForStats()
+    return self.ui.pagemap and self.ui.pagemap:wantsPageLabels()
+end
```

Comment:
May be move this one above :initData() (before it is used, just for context, rather than here in the middle of nowhere :)).